pipeline.libsonnet

```diff
@@ -75,9 +73,7 @@
         // TODO: For Docker, the name field should be registry and repository.
         name: matchArtifact.name,
         type: matchArtifact.type,
-        kind: matchArtifact.kind,
         [if 'artifactAccount' in matchArtifact then 'artifactAccount']: matchArtifact.artifactAccount,
-        customKind: if matchArtifact.kind == 'custom' then true else false,
```

Comment:
[This in deck](https://github.com/spinnaker/deck/blob/7791003a487a2f261db0c3f9af32e2081773f3dc/app/scripts/modules/core/src/domain/IArtifact.ts#L11-L13) does make it seem like they're both OK to delete.  I see `customKind` in orca, but only in test code.  Same for clouddriver.  There are still a few mentions in deck in non-test code, so ideally those would disappear first, but I also bet this is OK.  Would love someone to take a peek in deck and confirm.

Reply:
I'm not confident this is safe to remove

Reply:
@Chinikins any idea how this field is intended to be used?